CHANGELOG.md

```diff
@@ -52,6 +52,8 @@
 * refactor: Table class to fix phpstan errors by @kenjis in https://github.com/codeigniter4/CodeIgniter4/pull/8402
 * fix: typo in pager default_simple by @jasonliang-dev in https://github.com/codeigniter4/CodeIgniter4/pull/8407
 * refactor: improve Forge variable names by @kenjis in https://github.com/codeigniter4/CodeIgniter4/pull/8434
+* refactor: update entity to use interface for better extensibility by @colethorsen in https://github.com/codeigniter4/CodeIgniter4/pull/8325
```

Comment:
Thank you. But this is a changelog for v4.4.5, not v4.5.0, and this file is generated by GitHub.

Remove this, and add in the seciton "Others" in "Enhancements" in the following file:
https://github.com/codeigniter4/CodeIgniter4/blob/4.5/user_guide_src/source/changelogs/v4.5.0.rst#others-1